hack/test

```diff
@@ -153,6 +161,9 @@ function start_azurite {
 }
 
 function unit_pass {
+	# make sure backup operator can be built for each unit test run.
+	build_backup_operator
```

Comment:
build process shouldn't be put in unit test process.

Reply:
@hongchaodeng it is hack for now to make sure that backup-operator are being build every time or I can changed jenkin to run build pass.

Reply:
The jenkins runs `build_pass` in `jenkins-ci` job. Check the CI jobs configs

Reply:
good to know.